src/backoff.ts

```diff
@@ -105,16 +109,21 @@ export async function retry(
       return result;
     } catch (err) {
       originalError = err;
-      if (!predicate || !predicate(err, args)) {
-        throw err;
+      if (!predicate || !predicate(originalError, args)) {
```

Comment:
not sure of the use of the `originalError` variable. Seems that it is simply equivalent to `err`...

Reply:
it is, but I wanted both if conditions to be written the same way...


\#OCD